e2e-tests/reports/json/tmp/as-a-user.json

```diff
@@ -0,0 +1 @@
+[{"keyword":"Feature","type":"feature","description":"    I want to view a landing page with an option to authenticate\n    So that I can login to the trial","line":3,"name":"As a user","uri":"Can not be determined","tags":[{"type":"Tag","location":{"line":1,"column":1},"name":"@landingPage"},{"type":"Tag","location":{"line":1,"column":14},"name":"@arts-195"}],"elements":[{"keyword":"Scenario","type":"scenario","description":"    I want to view a landing page with an option to authenticate\n    So that I can login to the trial","name":"a trial has been launched with a configured name","tags":[{"name":"@landingPage","location":{"line":1,"column":1}},{"name":"@arts-195","location":{"line":1,"column":14}}],"id":"as-a-user;a-trial-has-been-launched-with-a-configured-name","steps":[{"arguments":[],"keyword":"Before","name":"Hook","result":{"status":"passed","duration":3000000},"line":"","match":{"location":"can not be determined with webdriver.io"}},{"arguments":[],"keyword":"Given","name":"User navigates to the landing page URL","result":{"status":"passed","duration":1099000000},"line":9,"match":{"location":"can not be determined with webdriver.io"}},{"arguments":[],"keyword":"When","name":"the landing page displays the name of the trial","result":{"status":"passed","duration":41000000},"line":10,"match":{"location":"can not be determined with webdriver.io"}},{"arguments":[],"keyword":"After","name":"Hook","result":{"status":"passed","duration":0},"line":"","match":{"location":"can not be determined with webdriver.io"}}]}],"id":"as-a-user","metadata":{"browser":{"name":"chrome","version":"87.0.4280.88"},"device":"Device name not known","platform":{"name":"osx","version":"Version not known"}}}]
```

Comment:
Can we do the same as with the allure reports here? Delete from the repo and add to gitignore

Reply:
I have added cucumber reports and allure reports to the .gitignore

<img width="1552" alt="Screenshot 2021-01-15 at 15 18 30" src="https://user-images.githubusercontent.com/31847267/104744572-eff2ee00-5744-11eb-9c0d-1911b1a7f866.png">